packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -78,9 +79,28 @@ mixin GestureBinding on BindingBase implements HitTestable, HitTestDispatcher, H
   static GestureBinding get instance => _instance;
   static GestureBinding _instance;
 
+  /// Determines whether the binding will record the input [ui.PointerData] to
+  /// the timeline.
+  ///
+  /// This flag is bypassed in release mode.
+  ///
+  /// See [serializePointerData] and [deserializePointerData].
+  bool recordInput = false;
```

Comment:
We usually call these `debugRecordInput` or somethign like that.

They also usually go into the `debug.dart` in their respective folder, and there should be a method in there that helps check these are not left in an unexpected state at the end of a test.

Reply:
I'm imagining this can be used both in debug mode and in profile mode. Not sure if they will produce some different record.

Reply:
Maybe @goderbauer could confirm, but I think we still call it debug in that case.

Reply:
@dnfield Sorry for any confusion. I meant that was what I thought when I wrote those codes. I've already changed it to debug and calling it `debugRecordInputs`